### ISSUE TITLE
God Mode send text does not send text

### ISSUE THREAD

**Issue by diwako** [IDI-Systems/acre2#1023]
**Mods (complete and add to the following information):**
- **Arma 3:** `1.98.146728 perf branch`
- **CBA:** `3.15.1.200418`
- **ACRE2:** `2.8.0.1037`

**Description:**
It is not possible to send messages to god mode groups, it will always say there are no units in the group

**Steps to reproduce:**
- Set up god mode, add yourself to a group
- send message to group

```sqf
[true, true] call acre_api_fnc_godModeConfigureAccess;
[[getPlayerUID player], 2, 0] call acre_sys_godmode_fnc_modifyGroup;

[{
    ["hello there!", 2] call acre_api_fnc_godModeSendText;
}, nil, 5] call cba_fnc_waitAndExecute;
```

**Expected behavior:**
Text "hello there!" will appear in the notification window

**Where did the issue occur?**
- Dedicated / Self-Hosted Multiplayer / Editor (Multiplayer)

**Additional context:**
Issue seems to stem from here
https://github.com/IDI-Systems/acre2/blob/1f363d002b337c8451f6a09ec6fc5b355aca9e5c/addons/sys_godmode/fnc_sendText.sqf#L31

`GVAR(targetUnits)` is only filled when speaking in a god mode channel, it will be emptied once you let go of the PTT button. It appears `_targetUnits` should be used instead.

**Comment by jonpas** [comment]
Nice find and correct fix. Can you open a PR? :)

**Comment by diwako** [comment]
Can I do it in October? :^)

**Comment by jonpas** [comment]
Fixed in #1026.
